src/sif/autocost.cc

```diff
@@ -716,25 +714,6 @@ class BusCost : public AutoCost {
                               const uint64_t current_time,
                               const uint32_t tz_index,
                               int& restriction_idx) const override;
-
-  /**
-   * Function to be used in location searching which will
-   * exclude and allow ranking results from the search by looking at each
-   * edges attribution and suitability for use as a location by the travel
-   * mode used by the costing method. It's also used to filter
-   * edges not usable / inaccessible by bus.
-   */
-  float Filter(const baldr::DirectedEdge* edge, const graph_tile_ptr& tile) const override {
```

Comment:
this and the others below were redundant, they were the same as autocost so i just let them use that one